nix/overlay.nix

```diff
@@ -60,15 +60,21 @@ in {
     version = "15.0";
     allowHigher = true;
   };
-  go = super.go_1_19;
+  go = super.go_1_20;
   clang = super.clang_15;
-  buildGoPackage = super.buildGo119Package;
-  buildGoModule = super.buildGo119Module;
+  buildGoPackage = super.buildGo120Package;
+  buildGoModule = super.buildGo120Module;
```

Comment:
I think i should revert the go upgrade here to 1_20 and do that separately.
Maybe that causes issues

Reply:
I agree.

Reply:
reverting go to 1.19 and getting rid of the `netgo` build flag for go-mobile fixed the issue